src/objects.h

```diff
@@ -808,9 +801,11 @@ extern void (* PrintPathFuncs[LAST_REAL_TNUM+1]) (
 
 /****************************************************************************
 **
-*F  SET_TYPE_DATOBJ( <obj>, <val> )  . . . . .  set the type of a data object
+*F  SetTypeDatobj( <obj>, <kind> ) . . . . . .  set the type of a data object
+**
+**  'SetTypeDatobj' sets the kind <kind> of the data object <obj>.
 */
-#define SET_TYPE_DATOBJ(obj,val)  (ADDR_OBJ(obj)[0] = (val))
+#define SetTypeDatObj(obj, type)  (ADDR_OBJ(obj)[0] = (type))
 
```

Comment:
Well, I forced myself to refrain from such things, as that mixes syncing with changing the code -- and any change can introduce regressions. So I'd prefer to do that in a second step.